TOMP-API.yaml

```diff
@@ -1251,25 +1249,25 @@ components:
           type: string
         geometry:
           $ref: '#/components/schemas/polygon'
-        opening-times:
+        openingTimes:
           description: the opening times of the facility
           type: array
           items:
-            $ref: '#/components/schemas/period'
+            $ref: '#/components/schemas/systemHours'
 
     polygon:
       type: object
       properties:
         points:
           type: array
           items:
-            $ref: '#/components/schemas/coordinates'
+            $ref: '#/components/schemas/coordinate'
```

Comment:
Why did you switch back?

Reply:
Merge problem? It should be coordinates.

Reply:
I guess so, I'll change it back :)